### ISSUE TITLE
Why no show_accuracy=True for Graph model?

### ISSUE THREAD

**Issue by audiofeature** [fchollet/keras#1744]
For a Sequential model, I can do

 model.fit(..., show_accuracy=True)

which will show the accuracy on the train set (and also validation set, if given). I tend to visualize these over the epochs.

For a Graph model, this option is missing. Why? Can it be easily added? After all they are both derived from the Model class...

**Comment by tboquet** [comment]
I think that the multiple output capability makes it difficult to add a `show_accuracy` parameter that will work for every situation. You could write a custom callback to calculate it, it's a little bit of work but hopefully it won't be very hard.
